### PR TITLE
[OCF-75] dockerskeleton: Switch to CentOS 7

### PR DESCRIPTION
**Pull request by goldmann**
https://issues.jboss.org/browse/OCF-75

### CODE REVIEW

File: docker/runtimes/dockerskeleton/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,23 @@
-# Dockerfile for docker skeleton (useful for running blackbox binaries, scripts, or Python 3 actions) .
-FROM openwhisk/dockerskeleton:1.1.0
+# Latest images from Mar 22nd
+FROM jboss/base@sha256:39bcf23f34ca58db0769121674d2a82aa4ea2ae9c956e280cb0ba1ef64c68b51
 
-# OpenShift compatibility
-RUN chgrp -R 0 /actionProxy \
-  && chmod -R g+rwX /actionProxy
+ARG OPENWHISK_RUNTIME_PHP_VERSION="dockerskeleton@1.2.0"
```
Comment (jcrossley3) [inline docker/runtimes/dockerskeleton/Dockerfile]: did you mean PHP here? s/PHP/PYTHON/ maybe?

**Review by jcrossley3 — COMMENTED** [review]
Thanks for the test!

File: docker/runtimes/dockerskeleton/Dockerfile (hunk 1 above)
Comment (goldmann) [reply docker/runtimes/dockerskeleton/Dockerfile]: It should be `RUNTIME`... Fixing.

Comment (goldmann) [reply docker/runtimes/dockerskeleton/Dockerfile]: Fixed!

**Pull request merged**